component/samples/ethereum/sampleDashBoard.bal

```diff
@@ -0,0 +1,103 @@
+//
+// Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import ballerina.net.http;
+import ballerina.util;
+import org.wso2.ballerina.connectors.ethereum;
+
+const string LATEST = "latest";
+
+// This is a simple dashboard for an Ethereum wallet
+public function main (string[] args) {
+
+    endpoint<ethereum:ClientConnector> ethereumConnector {
+        create ethereum:ClientConnector(args[0], {}, args[1], toInt(args[2]));
+    }
+
+    http:HttpConnectorError e;
+
+    println("Sample Ethereum Client");
+    println("------------------------------------------------------");
+
+    //get client version
+    var response, e = ethereumConnector.web3ClientVersion();
+    if (e != null) {
+        println(e);
+        return;
+    }
+    println("Web3 Client Version: " + response.result.toString());
+
```

Comment:
Shall we  have  a method similar to printResult (sample.bal) to print this result as well.